portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * chose a random element from an array
+ */
+function randomFromArray(arr){
+  randomChoice = Math.floor(Math.random() * arr.length)
+  return randomChoice
+}
+
 /**
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!', "I'm related to Paul Revere", "Never tickle a sleeping Dragon"];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const greeting = greetings[randomFromArray(greetings)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+let picShowing = false
```

Comment:
You can delete this since it is no longer used.

Reply:
Good catch!